manifests/params.pp

```diff
@@ -28,6 +28,7 @@
 
   $gunicorn_package_name = $facts['os']['family'] ? {
     'RedHat' => $facts['os']['release']['major'] ? {
+      '9' => 'python3-gunicorn',
       '8' => 'python3-gunicorn',
       default => 'python-gunicorn',
```

Comment:
We should flip this logic, since everything new is going to be `python3-gunicorn`.

```suggestion
      '7'     => 'python-gunicorn',
      default => 'python3-gunicorn',
```

Reply:
Done!
Thank you!